pkg/skaffold/constants/constants.go

```diff
@@ -75,8 +75,8 @@ const (
 type Phase string
 
 var (
-	Pod     latest.ResourceType = "pod"
-	Service latest.ResourceType = "service"
+	Pod     latest_v1.ResourceType = "pod"
```

Comment:
These are a little bit odd. Let try to follow up in another PR if ResourceType needs to be a version specifc

Reply:
Yes! Some latest_v1 is not accurate. I'm trying to keep this PR simple since it touches too many files.